### PR TITLE
optimize v2 test

### PR DESCRIPTION
**Pull request by bxy4543**
v2 Infra module has not been upgraded, and the test also temporarily shields infra test;
Because infra has not implemented V2 cluster to pull infra, and v2 Apply is compatible with v1, the test will not be upgraded to v2 for the time being.
Add debug mode to display debug logs for e2e tests.

### CODE REVIEW

File: test/suites/build/fixtures/cloud_build/Kubefile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM sealer-io/test:v1
+FROM sealer-io/kubernetes:v1.19.8
```
Comment (kakaZhou719) [inline test/suites/build/fixtures/cloud_build/Kubefile:1]: cloud build will push image automatically ,this action will update our base image ,so i suggested that we use the test base image.

File: test/suites/build/fixtures/cloud_build/imageList [hunk 2]
```diff
@@ -0,0 +1,4 @@
+quay.io/tigera/operator:v1.17.4
+docker.io/redis:6.2.5
+registry.cn-qingdao.aliyuncs.com/sealer-apps/debug:ubuntu
```
Comment (kakaZhou719) [inline test/suites/build/fixtures/cloud_build/imageList:3]: less docker image we add ,less time we cost

File: test/suites/apply/apply.go [hunk 3]
```diff
@@ -122,11 +123,11 @@ func GenerateClusterfile(clusterfile string) {
 }
 
 func SealerDeleteCmd(clusterFile string) string {
-	return fmt.Sprintf("%s delete -f %s --force", settings.DefaultSealerBin, clusterFile)
+	return fmt.Sprintf("%s delete -f %s --force -d", settings.DefaultSealerBin, clusterFile)
```
Comment (fanux) [inline test/suites/apply/apply.go:126]: What is -d for ?

**Review by kakaZhou719 — APPROVED** [review]

**Pull request merged**